resources/views/forum/draad.blade.php

```diff
@@ -25,12 +25,15 @@ class="btn btn-light" title="Toon statistieken">@icon('chart_line')</a>
 				<a title="Onderwerp toevoegen aan favorieten" class="btn btn-light post popup addfav"
 					 href="/menubeheer/toevoegen/favoriet">@icon('heart', 'heart_add')</a>
 				&nbsp;&nbsp;&nbsp;
-				@if($draad->isGevolgd())
-					<a href="/forum/volgenuit/{{$draad->draad_id}}" class="btn btn-light post ReloadPage volgenUit"
-						 title="Onderwerp niet meer volgen per email">@icon('email_go', 'email_delete')</a>
-				@elseif($draad->magVolgen())
-					<a href="/forum/volgenaan/{{$draad->draad_id}}" class="btn btn-light post ReloadPage volgenAan"
-						 title="Onderwerp volgen per email">@icon('email', 'email_add')</a>
+				@if($draad->magMeldingKrijgen())
+					<div class="btn-group">
+						<a href="/forum/meldingsniveau/{{$draad->draad_id}}/nooit" class="btn btn-light post ReloadPage melding-nooit @if($meldingsniveau == \CsrDelft\model\entity\forum\ForumDraadMeldingNiveau::NOOIT) active @endif"
```

Comment:
Voeg hier anders ook `is_null` toe, dan is er in ieder geval altijd eentje actief.

Reply:
Dat geeft dan alleen niet de actuele staat weer, dat hangt af van de instellingen van het lid: als hij geen meldingen voor een draadje heeft ingesteld en in instellingen melding bij vermelding aan heeft staan, krijgt hij wel een melding.

Ik kan wel het niveau gebruiken wat bepaald wordt i.c.m. de instellingen, maar dan is er geen manier meer om onderscheid te maken tussen ingestelde meldingen en standaardmeldingen.

Reply:
Dat is wel een goeie ja, had die even over het hoofd gezien.

Reply:
Kan wel een keuze zijn, maakt het wel simpel om per draadje te krijgen wat de meldinginstelling is